### PR TITLE
skopeo: 1.2.3 -> 1.3.0

### PR DESCRIPTION
**Pull request by saschagrunert**
###### Motivation for this change
Update to the latest release.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

cc @NixOS/podman @lewo

### CODE REVIEW

File: pkgs/development/tools/skopeo/default.nix [hunk 1]
```diff
@@ -31,8 +31,7 @@ buildGoModule rec {
 
   nativeBuildInputs = [ pkg-config go-md2man installShellFiles makeWrapper ];
 
-  buildInputs = [ gpgme ]
-  ++ lib.optionals stdenv.isLinux [ lvm2 btrfs-progs ];
+  buildInputs = [ gpgme ] ++ lib.optionals stdenv.isLinux [ lvm2 btrfs-progs ];
```
Comment (zowoq) [inline pkgs/development/tools/skopeo/default.nix:34]: Why is this included with the version bump?

Comment (saschagrunert) [reply pkgs/development/tools/skopeo/default.nix]: This was the auto formatter, I reverted that change.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 123780` at df7adc63 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>nix-prefetch-docker</li>
    <li>skopeo</li>
    <li>tests.trivial</li>
  </ul>
</details>

File: pkgs/development/tools/skopeo/default.nix (hunk 1 above)
Comment (zowoq) [reply pkgs/development/tools/skopeo/default.nix]: Are you using `nixpkgs-fmt`? Asking about this because we might start using it treewide soon and it shouldn't change multi-line into single-line.

Comment (saschagrunert) [reply pkgs/development/tools/skopeo/default.nix]: Yes, that's exactly how it behaved on my side too. I did the line-merge manually since it was still lower than 80 chars width

Comment (zowoq) [reply pkgs/development/tools/skopeo/default.nix]: Ah okay, sorry for the noise.
